detox/cli.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
```

Comment:
Why did you prefer putting this here and not inside `/local-cli` ?

Reply:
I can not put `.` as `commandDir` and I did not want to have this extra level of nesting 🤷‍♂️

Reply:
Why not?
```diff
-  "detox": "local-cli/detox.js"
+  "detox": "cli.js"
```

This is a part of the changes you made in this PR, why can't it be reverted?

Reply:
Because of [this part](https://github.com/wix/Detox/pull/1109/files#diff-aeb1fd007adaeb0ed56930b012da8ef1R11) in my changes. At least as far as I understand it. Feel free to try it, though.